theming/theme-package.rst

```diff
@@ -479,22 +476,22 @@ As a starting point we use this rules set:
      </replace>
 
      <!-- Left column -->
-     <rules if-content="//*[@id='portal-column-one']">
+     <rules css:if-content="#portal-column-one">
        <replace css:theme="#column1-container">
-           <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="sidebar">
+           <div class="col-xs-6 col-sm-3 sidebar-offcanvas">
```

Comment:
As I mentioned [here](https://github.com/plone/training/issues/174#issuecomment-159054729), because of plone/plonetheme.barceloneta#53